korean/coco/coco.md

```diff
@@ -0,0 +1,6 @@
+핫초코를 사러 나온 당신... 돈은 있나요? 핫초코를 어디서 살건가요? 
+
+돈이 없으면 뭘 어쩌겠어? [다시 방으로 돌아간다](../marshmallow.md) 집에서 잠이나 자!
+
+돈이 없으면 핫초코를...직접 찾아나서는거! [핫초코를 찾아 여행](../explore-outside/find-coco.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.